myapp/models.py

```diff
@@ -3,33 +3,36 @@
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
+
 # Create your models here.
 class Stock(models.Model):
-	symbol = models.CharField(max_length=12, primary_key=True)
-	name = models.CharField(max_length=64)
-	top_rank = models.IntegerField(null=True)
-	price = models.FloatField()
-	change = models.FloatField(null=True)
-	change_percent = models.FloatField()
-	market_cap = models.FloatField(null=True)
-	primary_exchange = models.CharField(null=True, max_length=32)
-
-	@classmethod
-	def add_to_watchlist(cls, profile, stock_symbol):
-		stock = cls.objects.get(symbol=stock_symbol)
-		profile.watchlist.add(stock)
-		profile.save()
-	
-	@classmethod
-	def remove_from_watchlist(cls, profile, stock_symbol):
-		stock = cls.objects.get(symbol=stock_symbol)
-		profile.watchlist.remove(stock)
-		profile.save()
+    symbol = models.CharField(max_length=12, primary_key=True)
+    name = models.CharField(max_length=64)
+    top_rank = models.IntegerField(null=True)
+    price = models.FloatField()
+    change = models.FloatField(null=True)
+    change_percent = models.FloatField(null=True)
+    market_cap = models.FloatField(null=True)
+    primary_exchange = models.CharField(null=True, max_length=32)
+    last_modified = models.DateTimeField(auto_now=True)
```

Comment:
Maybe we can also show the 'last modified' field on the relevant html template as well 👍